utility/src/main/java/org/oppia/util/parser/RepositoryGlideModule.kt

```diff
@@ -1,17 +1,22 @@
 package org.oppia.util.parser
 
 import android.content.Context
+import android.graphics.drawable.PictureDrawable
 import com.bumptech.glide.Glide
 import com.bumptech.glide.Registry
 import com.bumptech.glide.annotation.GlideModule
 import com.bumptech.glide.module.AppGlideModule
+import com.caverock.androidsvg.SVG
 import java.io.InputStream
 import org.oppia.util.caching.AssetRepository
 
 /** Custom [AppGlideModule] to enable loading images from [AssetRepository] via Glide. */
 @GlideModule
 class RepositoryGlideModule : AppGlideModule() {
   override fun registerComponents(context: Context, glide: Glide, registry: Registry) {
-    registry.prepend(ImageAssetFetcher::class.java, InputStream::class.java, RepositoryModelLoader.Factory())
+    registry.register(SVG::class.java, PictureDrawable::class.java, SvgDrawableTranscoder())
+      .prepend(SVG::class.java, SvgEncoder())
+      .append(InputStream::class.java, SVG::class.java, SvgDecoder())
+      .append(ImageAssetFetcher::class.java, InputStream::class.java, RepositoryModelLoader.Factory())
```

Comment:
I think as a whole what you put together seems reasonable. I think we can maybe simplify it a bit further:
- We could create a custom type to represent an OppiaImage (this is a good time to make use of Kotlin's sealed classes--we want exactly one subtype of OppiaImage that corresponds to bitmaps, and a second for SVGs)
- We would then need to update ImageAssetFetcher, ImageLoader, and GlideImageLoader to reference OppiaImage instead of Bitmap
- We would probably need to introduce a mechanism in OppiaImage to convert itself from/to a ByteArray to work correctly with the asset system
- Some of the changes you made below in UrlImageParser and maybe here will likely need to change if we pass around an OppiaImage instead of a Bitmap

The above changes help make SVGs more of a 'first class citizen" rather than being sort of tacked onto the bitmap loading pipeline. That may help provide some simplifications in the future, as well.

Reply:
Do we need to introduce OppiaImage in this PR itself.

Reply:
If you can solve all of the other comments without making these changes, I think they're fine as a follow-up work item. If you choose to go that route, please file an issue to track it, add a TODO in this file for that issue, and please make sure all of the details above are captured in the issue.

Reply:
> Thanks @veena14cs. I followed up to the comments I saw that had open questions--please let me know if I missed any or if you need additional input on anything.

@BenHenning I have addressed all of your comments. Please take a glance once and let me know if any further changes are required.

Reply:
I have created issue #1039  to track this and added todo comment in the code.

Reply:
Done.